bi-connectors/TableauConnector/src/manifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='opensearch_jdbc' superclass='jdbc' plugin-version='1.0' name='OpenSearch' version='18.1' min-version-tableau='2021.1'>
+<connector-plugin class='opensearch_jdbc' superclass='jdbc' plugin-version='1.0.1' name='OpenSearch' version='18.1' min-version-tableau='2021.1'>
```

Comment:
is this going to work on the verification side?

Reply:
Which verification side?

Reply:
Tableau CI will check this version matches signed taco file name version. Will let build team know that version changed.